modules/dashboard/test/TestPlan.md

```diff
@@ -33,14 +34,14 @@
     Clicking on this task should take you to the BVL statistics page, with the stats filtered according to the user's
     site (or without any filter if the user has 'Across all sites access candidates profiles' permission).
     [Automate Test on Travis_CI]
-12. Verify that if a user has 'Across all sites create and edit user accounts' permission, the number of pending
+12. Verify that if a user has 'Across all sites create and edit users' permission, the number of pending
```

Comment:
I believe 'user management' is the permission needed here. Consider changing the sentence to: Verify that if a user has 'Across all sites create edit users' and 'User management' permissions, the pending ...